doc/user/data/sql_funcs.yml

```diff
@@ -111,11 +111,14 @@
     description: "`x % y`"
 
   - signature: 'round(x: N) -> N'
-    description: "`x` rounded to the nearest whole number; halves are rounded up"
+    description: >-
+      `x` rounded to the nearest whole number.
+      If `N` is `real` or `double precision`, rounds ties to the nearest even number.
+      If `N` is `numeric`, rounds ties away from zero.
```

Comment:
These release note changes for numerics are not correct (the following run from this branch):

```
materialize=> select round(0.5::float8), round(0.5::numeric);
 round | round 
-------+-------
     0 |     0
```

Postgres:

```
mjibson=# select round(0.5::float8), round(0.5::numeric);
 round | round 
-------+-------
     0 |     1
```

Reply:
Oh, hah, great catch. I guess the documented behavior of decimals was wrong. Will address.

Reply:
Wait, no, this is just a nice footgun in Materialize. The default decimal scale is 0 so the decimal component of your input there just got dropped on the floor. Here's how it looks if one keeps their foot intact:

```
materialize=> create table t (f float8, n numeric(10, 5));
CREATE TABLE
materialize=> insert into t values (0.5, 0.5), (1.5, 1.5), (-0.5, -0.5), (-1.5, -1.5);
INSERT 0 4
materialize=> select f, round(f), round(n) from t;
  f   | round |  round   
------+-------+----------
  0.5 |     0 |  1.00000
 -0.5 |    -0 | -1.00000
  1.5 |     2 |  2.00000
 -1.5 |    -2 | -2.00000
(4 rows)

```

Maybe another bug in the way we handle rounding when converting to a numeric in the first place, but I think the release notes are actually ok as is.